openmc/surface.py

```diff
@@ -139,65 +152,84 @@ def create_xml_subelement(self):
 
         element.set("type", self._type)
         element.set("boundary", self._boundary_type)
-
-        coeffs = ''
-
-        for coeff in self._coeff_keys:
-            coeffs += '{0} '.format(self._coeffs[coeff])
-
-        element.set("coeffs", coeffs.rstrip(' '))
+        element.set("coeffs", ' '.join(map(str, self._coeffs)))
 
         return element
 
 
-
 class Plane(Surface):
+    """An arbitrary plane of the form :math:`Ax + By + Cz = D`.
+
+    Parameters
+    ----------
+    surface_id : int
+        Unique identifier for the surface
+    boundary_type : {'transmission, 'vacuum', 'reflective', 'periodic'}
+        Boundary condition that defines the behavior for particles hitting the
+        surface. Defaults to transmissive boundary condition where particles
+        freely pass through the surface.
```

Comment:
in the comments for ``XPlane``, ``YPlane`` and ``ZPlane`` below you referred to the object as a "plane" while here you refer to it as a "surface" (*e.g.*, "The 'A' parameter for the surface"). This is a super minor inconsistency I know, but is a quick fix